libs/durability/disk_failure_recovery_libs.py

```diff
@@ -272,12 +272,14 @@ def get_user_data_space_in_bytes(self, master_obj: LogicalNode, memory_percent:
             return True, 0
 
     @staticmethod
-    def perform_near_full_sys_writes(s3userinfo, user_data_writes, bucket_prefix: str) -> tuple:
+    def perform_near_full_sys_writes(s3userinfo, user_data_writes, bucket_prefix: str,
+                                     client: int) -> tuple:
```

Comment:
@kulrk Can you add default value as 10 for client.

Reply:
added